src/parser/statement.js

```diff
@@ -150,9 +154,17 @@ module.exports = {
    * ```
    */
   read_inner_statement: function () {
+    let attrs = [];
+    if (this.token === this.tok.T_ATTRIBUTE) {
+      attrs = this.read_attr_list();
```

Comment:
This line doesn't seem to be covered by the tests according to #749 (see https://github.com/glayzzle/php-parser/pull/749). Does that mean attributes in inner statements are already handled somewhere else?

Reply:
It seems you can declare class methods inside inner_statements, im just not sure how to write a test to test such a case.